subseas_flex_fcst/layout.py

```diff
@@ -22,12 +22,25 @@
 LIGHT_GRAY = "#eeeeee"
 
 #Initialization for start date dropdown to get a list of start dates according to files available
-startDates = predictions.cpt_starts_list(DATA_PATH,CONFIG["forecast_mu_file_pattern"],"\w{3}-\w{1,2}-\w{4}")
+startDates = predictions.cpt_starts_list(
+    DATA_PATH,
+    CONFIG["forecast_mu_file_pattern"],
+    CONFIG["start_regex"],
+    format_in=CONFIG["start_format_in"],
+    format_out=CONFIG["start_format_out"],
+)
+
 
 def app_layout():
 
     # Initialization
-    fcst_mu = predictions.sel_cpt_file(DATA_PATH,CONFIG["forecast_mu_file_pattern"],list(CONFIG["leads"])[0],startDates[-1])
+    fcst_mu = predictions.sel_cpt_file(
+        DATA_PATH,
+        CONFIG["forecast_mu_file_pattern"],
+        list(CONFIG["leads"])[0],
```

Comment:
yes, the leads are in the country-wise overriding config files. I pasted their content in introduction (even though I see that the indenting has gone doing that). The leads's have different children values (e.g. wk1 vs L1) so they can't be in the master config, since for instance L1 would not overwrite wk1 but add a new element to leads.

Reply:
Oh yeah. I suggest you commit those files to this repo for now. Eventually they'll move to country-specific repos.